src/orion/testing/__init__.py

```diff
@@ -147,7 +150,7 @@ def generate_benchmark_experiments_trials(
 
 @contextmanager
 def create_study_experiments(
-    exp_config, trial_config, algorithms, task_number, max_trial, n_workers=(1,)
+    state, exp_config, trial_config, algorithms, task_number, max_trial, n_workers=(1,)
```

Comment:
This function would not need to be a contextmanager anymore right? But in any case, let's leave turning it into a standard function for another PR otherwise it will break all the test diffs.

Reply:
done

Reply:
You adjusted all the `with create_study_experiments` as well? :open_mouth:

Reply:
yes